### DISCUSSION TITLE
Lambda is returning a vector instead of map

### DISCUSSION THREAD

**Discussion by agentfunk** [kwojcik-blockether/holy-lambda#79] [Q&A] [answered]
I can't seem to figure out why the default retriever is limited to string/bytes/map/future but doesn't "support" vector of maps (which is a valid json). I say support loosely because the response is actually returned (and works just fine for vectors) which is the right behavior but also prints a warning to stdout.  Here is the reference to the code: https://github.com/FieryCod/holy-lambda/blob/96b8902c374b6f0a106549e6a355afb6f630ab4f/modules/holy-lambda-default-retriever/src/fierycod/holy_lambda/retriever.cljc#L55

**Comment by kwojcik-blockether** [comment, marked as answer]
You're right. It should not trigger the warning. I got some unpublished changes to the retriever that I plan to release at the end of the week. Thank you for your input.

**Reply by agentfunk** [reply]
Thanks @FieryCod. Would be happy to help. Least I can do.

**Reply by kwojcik-blockether** [reply]
@agentfunk I have just released a new version of HL. This should fix the issue. Let me know if v0.6.6 works for you.
